app/tests/unit/pages/project/project-revision-create.test.tsx

```diff
@@ -0,0 +1,66 @@
+import "@testing-library/jest-dom";
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ProjectRevisionCreate } from "pages/cif/project-revision/[projectRevision]/project-revision-create";
+import PageTestingHelper from "tests/helpers/pageTestingHelper";
+import compiledprojectRevisionCreateNewQuery, {
+  projectRevisionCreateNewQuery,
+} from "__generated__/projectRevisionCreateNewQuery.graphql";
+
+const defaultMockResolver = {
+  Project() {
+    return {
+      pendingNewProjectRevision: "mock-revision-id",
+    };
+  },
+};
```

Comment:
relay already mocks things by default, if we're not checking for specific values we can just not pass a default query resolver